frontend/src/component/admin/network/NetworkTrafficUsage/NetworkTrafficUsagePlanSummary.tsx

```diff
@@ -31,94 +44,145 @@ const StyledCardDescription = styled(Box)(({ theme }) => ({
-                                condition={includedTraffic > 0}
-                                show={
+        <StyledContainerGrid container spacing={4}>
+            <StyledGrid item xs={5.5} md={5.5}>
+                <StyledContainer>
+                    <StyledColumnGrid item>
+                        <StyledCardTitleRow>
+                            <b>Number of requests to Unleash</b>
+                        </StyledCardTitleRow>
+                        <StyledCardDescription>
+                            <RowContainer>
+                                Incoming requests selected month{' '}
+                                <StyledNumbersDiv>
                                     <ConditionallyRender
-                                        condition={
-                                            usageTotal <= includedTraffic
-                                        }
+                                        condition={includedTraffic > 0}
                                         show={
-                                            <Badge color='success'>
-                                                {usageTotal.toLocaleString()}{' '}
-                                                requests
-                                            </Badge>
+                                            <ConditionallyRender
+                                                condition={
+                                                    usageTotal <=
+                                                    includedTraffic
+                                                }
+                                                show={
+                                                    <Badge color='success'>
+                                                        {usageTotal.toLocaleString()}{' '}
+                                                        requests
+                                                    </Badge>
+                                                }
+                                                elseShow={
+                                                    <Badge color='error'>
+                                                        {usageTotal.toLocaleString()}{' '}
+                                                        requests
+                                                    </Badge>
+                                                }
+                                            />
                                         }
                                         elseShow={
-                                            <Badge color='error'>
+                                            <Badge color='neutral'>
                                                 {usageTotal.toLocaleString()}{' '}
                                                 requests
                                             </Badge>
                                         }
                                     />
```

Comment:
Add 'neutral' to the ternary as well and you can get rid of this other conditional render with the repeated badge 😄